spec/fixtures/content_store/topical_event.json

```diff
@@ -24,6 +25,7 @@
       "alt_text": "Text describing the image"
     },
     "end_date": "2016-04-28T00:00:00+00:00",
+    "emphasised_organisations": [],
```

Comment:
Would it be possible to add a test for the presence of the organisation logos?  And also check that a non-emphasised organisation doesn't have a logo.

Reply:
I've pushed a new test for this in a separate commit, re-review would be fantastic!